src/envoy/http/authn/origin_authenticator.cc

```diff
@@ -92,7 +92,7 @@ bool OriginAuthenticator::run(Payload* payload) {
 
   // If none of the JWT triggered, origin authentication will be ignored, as if
   // it is not defined.
```

Comment:
Please update this comment.

Reply:
Updated the code, this doesn't need change anymore